client/boot/common.js

```diff
@@ -127,6 +127,10 @@ const oauthTokenMiddleware = () => {
 			'/connect',
 		];
 
+		if ( config.isEnabled( 'jetpack/pricing-page' ) ) {
```

Comment:
I'm wondering if we should check for Jetpack cloud as well.

Reply:
Yes. This is a good idea!